src/index.ts

```diff
@@ -0,0 +1,153 @@
+// 토, 일요일 정보는 포함하지 않음
+const KOREAN_HOLIDAYS: { [year: number]: { [month: number]: { [day: number]: boolean } } } = {
```

Comment:
```suggestion
const KOREAN_HOLIDAYS: { [year: number]: { [month: number]: Set<number> } } = {
```
month가 포함하는 일 수를 `Set` 또는 Array로 관리하는 것은 어떨까요?

Reply:
하루씩 더하거나 빼면서 계산하는게 이 모듈의 주 로직인데 Array로 하면 매번 배열에 있는 지 여부를 계산해야 하거나 코드가 복잡해질 거 같아서 Array는 고려하지 않았어요. 
Set으로 해볼 수는 있는데 이번 PR에서 하려면 며칠 더 걸릴 거 같네요 😅
추후 별도 PR에서 고려해봐도 될까요? @luaneyed

Reply:
@luaneyed Set 형태로 변경해봤습니다
https://github.com/croquiscom/korean-business-day/pull/1/commits/e40b0d2a52a054b2c501e1047077baffc77e836a

Reply:
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Set
Set이 낮은 버전의 브라우저에서는 지원하지 않을 듯 하네요. Set을 이용하는 커밋을 올렸다가 지웠습니다..

Reply:
브라우저에서 사용도 고려하는 것이면 target = es2017 이면 안 될 것 같긴 하네요. 브라우저용으로 모듈을 만들어보지 않아서 동작을 보장하려면 뭐가 필요할지 모르겠습니다

Reply:
우선 이정도로도 큰 문제는 없을 거 같아서 대화는 닫을 게요. 이 문제는 추후에 별도 PR에서 고민해보겠습니다. 감사합니다~ 🙂